jcl/src/java.base/share/classes/java/lang/Access.java

```diff
@@ -496,18 +496,22 @@ public String join(String prefix, String suffix, String delimiter, String[] elem
 		return String.join(prefix, suffix, delimiter, elements, size);
 	}
 
-/*[IF JAVA_SPEC_VERSION >= 20]*/
```

Comment:
Either leave this here, or repeat the `@Override` annotation for each variant of `ensureNativeAccess()`, otherwise the annotation applies to `isEnableNativeAccess(Module)` in Java 17, which is likely to be somewhat surprising.

Reply:
Line 509-511 applies to Java 21 (and out-of-service 20).
Java 8/11/17 have not `isEnableNativeAccess()`.

Reply:
Here is a snippet of what I see for Java 17:
```java
	public String join(String prefix, String suffix, String delimiter, String[] elements, int size) {
		return String.join(prefix, suffix, delimiter, elements, size);
	}

	@Override

	public boolean isEnableNativeAccess(Module mod) {
		return mod.implIsEnableNativeAccess();
	}
```
Notice the `@Override` annotation with blank lines before and after it (sourced from line 499 here). It's reasonable to add that annotation to `isEnableNativeAccess(Module)`, but this is not the way to do it.

Reply:
Ah, I misread it.
Moved `@Override` into each variant of `ensureNativeAccess()`.